Examples/ExamplesApp/Examples.swift

```diff
@@ -56,21 +56,31 @@ struct Examples: View {
     }
     
     static func makeCategories() -> [Category] {
-        let common: [Category] = [
-            .geoview,
-            .views
-        ]
 #if os(iOS) && !targetEnvironment(macCatalyst)
-        return [.augmentedReality] + common
+        [.accessories, .augmentedReality, .data, .other]
 #else
-        return common
+        [.accessories, .data, .other]
 #endif
     }
 }
 
 @MainActor
 extension Category {
+    /// Components that can be used with any geoview.
+    static var accessories: Self {
+        .init(
+            name: "Accessories",
+            examples: [
+                Example("Basemap Gallery", content: BasemapGalleryExampleView()),
```

Comment:
I moved Basemap Gallery into this category since [my original proposal](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/issues/992#issuecomment-2762671053) since it can be initialized without specific data on any geoview.